tethys_cli/site_commands.py

```diff
@@ -5,213 +5,166 @@
+    site_parser.add_argument('--home-page-template', dest='home_page_template',
+                             help='Path to alternate Home page template (will replace Home page template entirely). '
+                                  'The template must be located within a valid templates directory, such as in a '
+                                  'Tethys app, Tethys extension, or Django app.')
+    site_parser.add_argument('--apps-library-template', dest='apps_library_template',
+                             help='Path to alternate Apps Library page template (will replace Apps Library page '
+                                  'template entirely). The template must be located within a valid templates '
+                                  'directory, such as in a Tethys app, Tethys extension, or Django app.')
+    site_parser.add_argument('--login-page-template', dest='login_page_template',
+                             help='Path to alternate portal login page template (will replace login page template '
+                                  'entirely). The template must be located within a valid templates directory, such '
+                                  'as in a Tethys app, Tethys extension, or Django app.')
+    site_parser.add_argument('--register-page-template', dest='register_page_template',
+                             help='Path to alternate portal registration (or signup) page template (will replace '
+                                  'signup page template entirely). The template must be located within a valid '
+                                  'templates directory, such as in a Tethys app, Tethys extension, or Django app.')
+    site_parser.add_argument('--user-page-template', dest='user_page_template',
+                             help='Path to alternate user profile page template (will replace user page '
+                                  'template entirely). The template must be located within a valid templates '
+                                  'directory, such as in a Tethys app, Tethys extension, or Django app.')
+    site_parser.add_argument('--user-settings-page-template', dest='user_settings_page_template',
+                             help='Path to alternate user settings (i.e. edit) page template (will replace settings '
+                                  'page template entirely). The template must be located within a valid templates '
+                                  'directory, such as in a Tethys app, Tethys extension, or Django app.')
+    site_parser.add_argument('-d', '--restore-defaults', dest='restore_defaults', action='store_true',
                              help='Restores the sites default values.')
-    site_parser.add_argument('-f', '--from-file', type=str, help='Load site content from portal_config.yml file.')
+    site_parser.add_argument('-f', '--from-file', action='store_true', default=False,
+                             help='Load site content from portal_config.yml file.')
```

Comment:
Our Docker build will need to be updated in at least two places to account for these changes:

1. The `tethyscore.sls` script will need to be updated (overhauled) accordingly. See: https://github.com/tethysplatform/tethys/blob/master/docker/salt/tethyscore.sls#L157
2. The `Dockerfile` environment variables. See: https://github.com/tethysplatform/tethys/blob/master/Dockerfile#L57